### PR TITLE
[opengl/egl-registry] update and add pc file

### PR DESCRIPTION
**Pull request by Neumann-A**
pc file from cygwin packaging

### CODE REVIEW

File: ports/egl-registry/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "egl-registry",
-  "version-string": "2020-02-20",
-  "port-version": 1,
+  "version-string": "2021-11-23",
```
Comment (JonLiu1993) [inline ports/egl-registry/vcpkg.json:3]:
```suggestion
  "version-date": "2021-11-23",
```

File: ports/glad/portfile.cmake [hunk 2]
```diff
@@ -112,17 +112,17 @@ string(REPLACE ";" "," GLAD_API "${GLAD_API}")
...
+    "${CURRENT_INSTALLED_DIR}/share/opengl/gl.xml"
+    "${CURRENT_INSTALLED_DIR}/share/opengl/glx.xml"
+    "${CURRENT_INSTALLED_DIR}/share/opengl/wgl.xml"
+    DESTINATION "${SOURCE_PATH}/glad/files"
 )
 
 vcpkg_cmake_configure(
-    SOURCE_PATH ${SOURCE_PATH}
+    SOURCE_PATH "${SOURCE_PATH}"
     PREFER_NINJA
```
Comment (JonLiu1993) [inline ports/glad/portfile.cmake:126]:
```suggestion
```

File: ports/glad/portfile.cmake [hunk 3]
```diff
@@ -143,7 +143,7 @@ vcpkg_cmake_install()
 vcpkg_copy_pdbs()
 vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/glad)
 
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include ${CURRENT_PACKAGES_DIR}/include/KHR)
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include ${CURRENT_PACKAGES_DIR}/include/EGL)
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include ${CURRENT_PACKAGES_DIR}/include/KHR")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include ${CURRENT_PACKAGES_DIR}/include/EGL")
```
Comment (JonLiu1993) [inline ports/glad/portfile.cmake:147]:
```suggestion
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include" "${CURRENT_PACKAGES_DIR}/include/KHR")
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include" "${CURRENT_PACKAGES_DIR}/include/EGL")

```

File: ports/glad/vcpkg.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "glad",
   "version": "0.1.34",
-  "port-version": 1,
+  "port-version": 3,
```
Comment (JonLiu1993) [inline ports/glad/vcpkg.json:4]:
```suggestion
  "port-version": 2,
```

File: ports/opengl-registry/vcpkg.json [hunk 5]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "opengl-registry",
-  "version-string": "2020-03-25",
-  "port-version": 1,
+  "version-string": "2021-11-17",
```
Comment (JonLiu1993) [inline ports/opengl-registry/vcpkg.json:3]:
```suggestion
  "version-date": "2021-11-17",
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/glad/vcpkg.json (hunk 4 above)
Comment (Neumann-A) [reply ports/glad/vcpkg.json:4]: yeah that is what I also thought but for some reason there is already a port-version 2 in the registry.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**